core/store/models/parse_randomness_request.go

```diff
@@ -36,6 +36,10 @@ func (parseRandomnessRequest) parseJSON(log Log) (js JSON, err error) {
 		"jobID": parsedLog.JobID.Hex(),
 		// Address of consuming contract which initially made the request
 		"sender": parsedLog.Sender.Hex(),
+		// Hash of the block in which this request appeared
```

Comment:
How about unserializing this into something like a RandomnessRequest with methods for accessing blockHash, blockNum etc?

Reply:
Do you mean as opposed to JSON? The `Random` adapter, and the adapter framework generally, requires a JSON input.